providers/implementations/ciphers/ciphercommon_hw.c

```diff
@@ -16,6 +17,9 @@
 int cipher_hw_generic_cbc(PROV_CIPHER_CTX *dat, unsigned char *out,
                           const unsigned char *in, size_t len)
 {
+    if (!provider_is_running())
```

Comment:
I thought you said you weren't going to put it in the update.. If this isn't an update I will eat my hat (and I don't have a hat).